test/fix-paths-orchestrator-task/in/main/other/keboola.orchestrator/orchestrator/phases/001-phase-1/001-task-1/task.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "Task 1",
+  "task": {
+    "mode": "run",
+    "configPath": "extractor/ex-generic-v2/old-name"
```

Comment:
Zacnime testom.
V `in`: mame v `Task` cestu ku configu, ktory bude premenovany.